apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx

```diff
@@ -138,15 +142,19 @@ export function TransferNFTForm({
 							</div>
 						</Content>
 						<Menu stuckClass="sendCoin-cta" className="w-full px-0 pb-0 mx-0 gap-2.5">
-							<Button
-								type="submit"
-								variant="primary"
-								loading={transferNFT.isLoading}
-								disabled={!isValid}
-								size="tall"
-								text="Send NFT Now"
-								after={<ArrowRight16 />}
-							/>
+							{isAccountWriteLocked ? (
+								<UnlockAccountButton account={activeAccount} />
```

Comment:
might be useful to create a component / Button that handles this logic so that we can re-use it without having to duplicate the logic for checking `isAccountWriteLocked` everywhere

Reply:
Yeah good point, I will add a new component for this